e2e/cypress/tests/integration/messaging/emoji_gender_spec.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login as test user and visit town-square
+        cy.apiInitSetup({loginAfter: true}).then(({offTopicUrl}) => {
+            cy.visit(offTopicUrl);
+        });
+    });
+
+    it('MM-T1539 Gendered emojis are rendered with the correct gender', () => {
+        // # Post a man-gesturing-ok emoji
+        cy.postMessage('🙆‍♂️');
+
+        // # Assert posted emoji was rendered as man
+        cy.getLastPost().find('.emoticon').should('have.attr', 'title', ':man-gesturing-ok:');
```

Comment:
Hi @sk409 ,
Thank you for the changes.
Would you like to add another emoticon `woman-gesturing-ok` to verification?
Also, please explore if we can use `cy.findByTitle` method instead of line 25.

Reply:
@yasserfaraazkhan 
Thank you for your review!
I added `soman-gestureing-ok` test and used `cy.findByTitle`.